vulcanai2/models/cnn.py

```diff
@@ -74,9 +74,9 @@ def _create_network(self):
     def _create_classification_layer(self, dim):
         self.network_tails = nn.ModuleList([DenseUnit(dim, out_d) for out_d in self.out_dim])
 
-    def forward(self, x):
+    def _forward(self, x):
```

Comment:
don't we need to have a `forward` function instead of the `_forward`? probably doesn't make much of a difference.